packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -56,6 +56,7 @@ export function HiddenSelect<T>(props: HiddenSelectProps<T>) {
     return (
       <VisuallyHidden aria-hidden="true">
         <input
+          type="text"
```

Comment:
I assume this came from a lint/test failure?

Reply:
the accessibility tree only recognizes it as a implicit role 'textbox' if it has the type text

this is observed in https://github.com/adobe-private/react-spectrum-v3/pull/424/files#diff-d09785ff8126fba708e5c25fc755e956R1443
`let hiddenInput = getByRole('textbox', {hidden: true});`